src/core/geometry/qgsgeometryutils.h

```diff
@@ -789,6 +789,16 @@ class CORE_EXPORT QgsGeometryUtils
      */
     static double triangleArea( double aX, double aY, double bX, double bY, double cX, double cY ) SIP_HOLDGIL;
 
+    /**
+     * Given the line (\a x1, \a y1) to (\a x2, \a y2) and a point (\a px, \a py) returns the percentage
```

Comment:
```suggestion
     * Given the line (\a x1, \a y1) to (\a x2, \a y2) and a point (\a px, \a py) returns the fraction
```

percentage would imply a return value between 0 and 100